user-guide/Advanced_Functionality/Databases/Elasticsearch_database/Manually_Connecting_DMA_to_Elasticsearch_Cluster.md

```diff
@@ -0,0 +1,36 @@
+---
+uid: Manually_Connecting_DMA_to_Elasticsearch_Cluster
+---
+# Manually connecting a DMA to an Elasticsearch cluster
+
+You can connect a DataMiner Agent to an existing Elasticsearch cluster by manually adding a *DataBase* tag to the *DB.xml* file, as detailed below.
+
```

Comment:
From 10.3.3 onwards, it is recommended to use the Cube UI, documentation should be updated then.